app/controllers/unattended_controller.rb

```diff
@@ -190,21 +189,6 @@ def allowed_to_install?
     (@host.build || @spoof || Setting[:access_unattended_without_build]) ? true : head(:method_not_allowed)
   end
 
-  # Cleans Certificate and enable autosign. This is run as a before_action for provisioning templates.
-  # The host is requesting its build configuration so I guess we just send them some text so a post mortum can see what happened
-  def handle_ca
-    # The reason we do it here is to minimize the amount of time it is possible to automatically get a certificate
-
-    # We don't do anything if we are in spoof mode.
-    return true if @spoof
```

Comment:
No longer needed ?

Reply:
Yeah, this can safely be removed. `@spoof` is true when the host templates are previewed by a user and not accessed by a host that wants to install. So it makes sense to not run any orchestration in preview mode.